website/client/src/components/tasks/task.vue

```diff
@@ -979,7 +979,9 @@ export default {
       // Prevent clicking on a link from opening the edit modal
       const target = e.target || e.srcElement;
 
-      if (target.tagName === 'A') return; // clicked on a link
+      for (let element = target; element.className === undefined || element.className.indexOf('task-clickable-area') === -1; element = element.parentNode) {
```

Comment:
Thanks @fmaximus , I think instead of using `element.className` and checking for `undefined` using `classList` and `.contains` should be easier to read https://developer.mozilla.org/en-US/docs/Web/API/Element/classList https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList 

I'd also add a comment explaining what the for loop does and why it's necessary